test/cmdlineargs.jl

```diff
@@ -2,22 +2,29 @@
 
 import Libdl
 
-catcmd = `cat`
-echocmd = `echo`
-if Sys.iswindows()
-    busybox = joinpath(Sys.BINDIR, "busybox.exe")
-    havebb = try # use busybox-w32 on windows
-        success(`$busybox`)
-        true
-    catch
-        false
-    end
-    if havebb
-        catcmd = `$busybox cat`
-        echocmd = `$busybox echo`
+# helper function for passing input to stdin
+# and returning the stdout result
+function writereadpipeline(input, exename)
```

Comment:
This functionality seems generally useful enough that we might want something like it in Base.

For example, maybe we could have an `EchoPipe(input)` that you could pass to `pipeline` to use a string as input?

Reply:
`pipeline(exename, stdin=IOBuffer(input))` would be good for this (and would be meaningful for use in either direction) – it just needs someone to make it happen.